pom.xml

```diff
@@ -191,7 +191,7 @@
     <maven-javadoc-plugin.version>3.1.1</maven-javadoc-plugin.version>
     <maven-shade-plugin.version>3.2.0</maven-shade-plugin.version>
     <maven-source-plugin.version>2.2.1</maven-source-plugin.version>
-    <maven-surefire-plugin.version>2.21.0</maven-surefire-plugin.version>
+    <maven-surefire-plugin.version>3.0.0-M5</maven-surefire-plugin.version>
```

Comment:
Please use 3.0.0-M4
Unfortunately M5 has some known problems about interactive development (basically it doesn't flush the output often  enough)  

Can you also point which problem did you see with Surefire?  And how it is related to RocksDB?

Reply:
Hi @eolivelli ,
Using 3.0.0-M4, 'Integration Tests' checks are failing again. It seems that it is not compatible. 
Refer: https://github.com/apache/bookkeeper/pull/2379/checks?check_run_id=949575926
Please share your opinion.